src/api.js

```diff
@@ -40,12 +41,16 @@ import {
 
 import DICOMwebClient from 'dicomweb-client/build/dicomweb-client.js'
 
+
 function _geometry2Scoord3d(geometry, pyramid) {
   const type = geometry.getType();
   if (type === 'Point') {
     let coordinates = geometry.getCoordinates();
     coordinates = _geometryCoordinates2scoord3dCoordinates(coordinates, pyramid);
-    return new Point(coordinates);
+    return new Point({
+      coordinates,
+      referencedFrameOfReferenceUID: pyramid[pyramid.length-1].frameOfReferenceUID
```

Comment:
I would move `pyramid[pyramid.length-1].frameOfReferenceUID`  to after line 46 and extract as a constant since it is the same for all geometries.

`const referencedFrameOfReferenceUID = pyramid[pyramid.length-1].frameOfReferenceUID `

```
return new Polygon({
      coordinates,
      referencedFrameOfReferenceUID
    });

...
return new Polyline({
      coordinates,
      referencedFrameOfReferenceUID
    });
...
etc
```

Reply:
Addressed with 707825a858700025c7520e7d2678099f3edc9d35